prboom2/INSTALL

```diff
@@ -90,6 +73,33 @@ Basic Installation
      See the README file for information about getting the Timidity
      patches needed for music support, and more.
 
+     
```

Comment:
Now this part is complete obsolete and irrelevant. We should not document building of the original project here which this fork is going to replace.

Reply:
Please remove the paragraph about building from SVN.